.wordpress-org/blueprints/blueprint.json

```diff
@@ -0,0 +1,27 @@
+{
+	"landingPage": "/wp-admin/options-reading.php",
+	"preferredVersions": {
+		"php": "7.4",
+		"wp": "6.4"
```

Comment:
Thinking we should set this to `latest` so we're not having to constantly bump this version